blogs/tests/test_templatetags.py

```diff
@@ -61,14 +54,14 @@ def test_feed_list(self):
             summary='',
             pub_date=now(),
             url='path/to/foo',
-            feed=f1
+            feed=f2
```

Comment:
The second feed never got a blog entry, so this test was only testing that items from f1 were returned. It seems to me the intention was to test that entries of both feeds are returned.